src/main/java/com/google/api/codegen/transformer/ruby/RubyImportSectionTransformer.java

```diff
@@ -95,6 +109,26 @@ public ImportSectionView generateImportSection(
     return filenames;
   }
 
+  private List<ImportFileView> generateTestStandardImports() {
+    return ImmutableList.of(createImport("minitest/autorun"), createImport("minitest/spec"));
+  }
+
+  private List<ImportFileView> generateTestAppImports(Model model, GapicProductConfig apiConfig) {
+    ImmutableList.Builder<ImportFileView> imports = ImmutableList.builder();
+    SurfaceNamer namer = new RubySurfaceNamer(apiConfig.getPackageName());
+    for (Interface service : new InterfaceView().getElementIterable(model)) {
+      imports.add(createImport(namer.getServiceFileName(apiConfig.getInterfaceConfig(service))));
+    }
+
+    // The protobuf imports need to be loaded before the client is initialized.
+    for (Interface service : new InterfaceView().getElementIterable(model)) {
```

Comment:
I was thinking it would look better to have the pb files and the service files separated. On second thought I think it would be better to be combined. Fixed.